config/www/cgi/text.py

```
@@ -0,0 +1,139 @@
+def load_sessions():
+    sessions = {}
+    if os.path.exists(SESSION_FILE):
+        with open(SESSION_FILE, "r") as file:
+            reader = csv.reader(file)
+            for row in reader:
+                if len(row) == 2:
+                    sessions[row[0]] = row[1]  # session_token: text
+    return sessions
+
+# Function to save session data to CSV
+def save_sessions(sessions):
```

Ruff

Missing return type annotation for public function `save_sessions`

Add return type annotation: `None`

(ANN201)